site/_scss/blocks/_share-button.scss

```diff
@@ -0,0 +1,25 @@
+share-button {
+  border-radius: 50%;
+  display: block;
+  height: 32px;
+  width: 32px;
+
+  &:hover {
+    background: var(--color-bg-shade);
+  }
+
+  .share-button__link {
+    display: inline-block;
+    height: 32px;
+    padding-left: 5px;
+    width: 28px;
```

Comment:
I'm confused, this still has the issue Rob mentioned, I can hover the circle without it being clickable. Can you just make this `width: 100%` without padding and set `text-align: center` or use Flex or something?

Reply:
Please try now